### PR TITLE
fix: tickets fetched at each step of wizard

### LINKED ISSUE #3409: Tickets are required for publishing event error even when tickets are present
**Describe the bug**

"Tickets are required for publishing event" is popping up even when tickets are present.
Error: 
image

Tickets : 
image

**Additional context**

Working on it

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #3409 

#### Changes proposed in this pull request:

- add 'tickets' property to be fetched if not present

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/mixins/event-wizard.js [hunk 1]
```diff
@@ -48,7 +48,9 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
   async saveEventData(propsToSave = []) {
     const event = this.get('model.event');
     const data = {};
-
+    if (!propsToSave.includes('tickets')) {
+      propsToSave.push('tickets');
```
Comment (CosmicCoder96) [inline app/mixins/event-wizard.js]: instead of this, just add it where the function is being called.

Comment (uds5501) [reply app/mixins/event-wizard.js]: @CosmicCoder96 Followed the suggestion, please review now

**Comment by uds5501** [comment]
@shreyanshdwivedi @mrsaicharan1 @CosmicCoder96 @kushthedude Please review this

**Comment by uds5501** [comment]
@kushthedude Please merge this

**Pull request merged**